types/bn.js/bn.js-tests.ts

```diff
@@ -16,3 +16,11 @@ const red = bn.toRed(ctx);
 const newRed = red.redAdd(new BN(1));
 newRed.cmp(bn);
 newRed.fromRed();
+
+const expected = new BN(0x4020);
+const actualArray = new BN([0x40, 0x20]);
+const actualUint8Array =  new BN(new Uint8Array([0x40, 0x20]));
+const actualString = new BN('0x4020');
+if (!actualArray.eq(expected)) throw new Error(`${actualArray} does not equal ${expected}`);
```

Comment:
Correct me if I'm wrong, not no one ever runs those files. They are tests for the types, not the behaviour and therefore any value equality checks are useless.

Reply:
I believe the npm run test command will execute them?  I put the test in to ensure that `BN` would actually accept a Uint8Array as a parameter and process it properly.  It is easy to author a type definition that doesn't align with reality, and this test ensures that the type definition aligns with reality.

Reply:
Theck the main README for this repo.
```
foo-tests.ts This contains sample code which tests the typings. This code does not run, but it is type-checked.
```
I'd like to have those run, but I think that in the current situation - it's dead code and it's misleading.

Reply:
A convincing argument.  Fixed.